app/overrides/remove_channel_from_order_view.rb

```diff
@@ -0,0 +1,3 @@
+Deface::Override.new(virtual_path: 'spree/admin/shared/_order_summary',
+  name: 'remove_channel_from_order_view',
+  remove: ".list-group-item.d-flex.justify-content-between.align-items-centerd")
```

Comment:
This removes all list elements. I've seen :first but no idea how to do this. The possible way to do this is to remove two elements which have id and then add_attribute padding-top/bottom 0 ;)

Reply:
Again - using ?jQuery it would be easy

Reply:
d in the end is added in purpose

Reply:
This is brute force https://github.com/TomaszKot11/ror-junior/commit/14d3ee1d051a14e3bab1fc2f4f2fb97602f4ccb7